narwhals/_arrow/series.py

```diff
@@ -70,6 +118,35 @@ def __mul__(self, other: Any) -> Self:
         other = validate_column_comparand(other)
         return self._from_native_series(pc.multiply(self._native_series, other))
 
+    def __truediv__(self, other: Any) -> Self:
+        pc = get_pyarrow_compute()
+        ser = self._native_series
+        other = validate_column_comparand(other)
+        return self._from_native_series(pc.divide(ser, other))
+
+    def __floordiv__(self, other: Any) -> Self:
+        pc = get_pyarrow_compute()
+        ser = self._native_series
+        other = validate_column_comparand(other)
+        return self._from_native_series(pc.floor(pc.divide(ser, other)))
```

Comment:
Need to double check behavior for negative numbers

Reply:
pandas does some crazy stuff here

https://github.com/pandas-dev/pandas/blob/262fcfbffcee5c3116e86a951d8b693f90411e68/pandas/core/arrays/arrow/array.py#L124-L154